app/components/repo-actions.js

```diff
@@ -59,16 +59,24 @@ export default Component.extend({
     }
   }),
 
+  showPriorityBtn: computed('showPriority', function () {
+    if (this.showPriority === false) {
+      return false;
+    }
+
+    return true;
+  }),
```

Comment:
I don't see any need in this property. It could just initialize with default:
```js
  showPriority: true,
```

Then external assignment will just overwrite this property with a new value.

Does it make sense to you?